src/main/java/com/onarandombox/MultiverseCore/MultiverseCore.java

```diff
@@ -350,9 +350,14 @@ public void onEnable() {
      * Initializes the buscript javascript library.
      */
     private void initializeBuscript() {
-        buscript = new Buscript(this);
-        // Add global variable "multiverse" to javascript environment
-        buscript.setScriptVariable("multiverse", this);
+        try {
+            buscript = new Buscript(this);
+            // Add global variable "multiverse" to javascript environment
+            buscript.setScriptVariable("multiverse", this);
+        } catch (NullPointerException e) {
+            buscript = null;
+            Logging.warning("Buscript failed to load! The script command will be disabled!");
+        }
```

Comment:
This could have implications for MVPortals.

Reply:
The issue that this addresses is rather obscure, I was hoping you could take a look at it.

Anyways, if you'd like to see this in MV Portals, let me know.

Reply:
I just mean that MV Portals probably expects this to not ever be null.

Reply:
I checked, and you're right. Would you like me to open a PR to take care of it?

Reply:
Sure. Did you do this already?

Reply:
Not yet, doing it now.

Reply:
We still need a fix for Portals to deal with this.